includes/discovery/vrf.inc.php

```diff
@@ -37,20 +48,19 @@
 
         $rds = trim($rds);
 
-        $descrs = snmp_walk($device, $descrs_oid, '-Osqn', $vpnmib, null);
-        $ports  = snmp_walk($device, $ports_oid, '-Osqn', $vpnmib, null);
+        if ($vpnmib != 'CISCO-VRF-MIB') {
```

Comment:
Seems like you can just check `$descrs_oid` is set here rather than the mib used